UndertaleModTool/Converters/UndertaleCachedImageLoader.cs

```diff
@@ -101,7 +93,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 
             if (tileRectList is not null)
             {
-                Rectangle rect = new(texture.SourceX, texture.SourceY, texture.SourceWidth, texture.SourceHeight);
+                Rect rect = new(texture.SourceX, texture.SourceY, texture.SourceWidth, texture.SourceHeight);
```

Comment:
It's this:
![image](https://github.com/Miepee/UndertaleModTool/assets/45438625/6fdd2a18-e67c-477f-b242-d91978f2f60a)

Reply:
I could probably have used `SKRectI`, but then I will have to slightly rewrite `CreateSpriteBitmap()`, because it changes `rect.Width` and `rect.Height`, and `SKRectI` properties are read-only.

Reply:
Ah. well this is currently in the windows only GUI, so I dont care that much. just couldnt immediately see from where that struct was coming from.